mkosi/config.py

```diff
@@ -4909,6 +4926,9 @@ def summary(config: Config) -> str:
       Kernel Modules Initrd Exclude: {line_join_list(config.kernel_modules_initrd_exclude)}
  Kernel Modules Initrd Include Host: {yes_no(config.kernel_modules_initrd_include_host)}
 
```

Comment:
Please drop this newline as well

Reply:
Same as before:

https://github.com/systemd/mkosi/blob/c2b7730ea0246285149d924c6d467808981ded2e/mkosi/config.py#L4902-L4910